SafeNeighbourhood/Classes/NeighborhoodData.cs

```diff
@@ -0,0 +1,81 @@
+﻿// <auto-generated />
+//
+// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
+//
+//    using QuicktypeNeighborhood;
+//
+//    var neighborhoodData = NeighborhoodData.FromJson(jsonString);
+
+namespace QuicktypeNeighborhood
```

Comment:
I created this Class based on the Json generated by a new datasource I was able to create on data.cincinnati-oh.gov that summarizes the cleanliness complaints of each neighborhood in Cincinnati.  Saved it as a view on the data.cincinnati-oh.gov website, which allows me to call it as a brand new Json.  The neighborhood field in this dataset matches the neighborhood field in the vacant homes Json previously used in the project.

Reply:
This is really cool.  I'd like to chat a bit before class and see how you did this!